pages/index.html

```diff
@@ -4,9 +4,37 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Personal Portfolio Website</title>
+  <link href="/styles/index_style.css" rel="stylesheet"/>
 </head>
+
 <body>
+  <header>
+    <img src="/assets/logo.png" alt="name logo" class="logo">
+    <nav>
+      <ul>
+        <li><a href="/pages/index.html">HOME</a></li>
+        <li><a href="/pages/about.html">ABOUT</a></li>
+        <li><a href="/pages/portfolio.html">PORTFOLIO</a></li>
+      </ul>
+    </nav>
+  </header>
   
+  <section class="intro">
+    <h1>Hi, I'm Tinghui <span>Jiang</span></h1>
+    <p> This is my official website. I am learning to become
+      <br> a software developer. I am always interested in learning new things.
+    </p>
```

Comment:
Try to avoid using `br`. We can use styling instead to have text wrap, which allows our page to resize better.